version.json

```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.0.0-rc.6",
+  "version": "1.0.0-rc.7",
```

Comment:
Assuming we get this merged eventually, we'd want to separate the version bump into a distinct commit.